.gitlab-ci.yml

```diff
@@ -19,6 +19,21 @@ cppcheck:
   tags:
     - docker
 
+flake8:
+  stage: build
+  image: pipelinecomponents/flake8:latest
+  variables:
+      FLAKE8_FILES: "tests/capi.py"
+  script:
+    - flake8 --verbose $FLAKE8_FILES
+
+flake8-diff:
+  stage: build
+  image: pipelinecomponents/flake8:latest
+  script:
+    - apk add git
```

Comment:
I'm surprised this works without `--update`, unless the image already contains a cache.
I'm not too familiar with apk though.

Another option you might want to use regardless is `--no-cache`.

Reply:
I guess they didn't clear the cache.

But indeed, going forward --update --no-cache is needed, because the cached reference may no longer be available.